pkg/cmd/repo/create/create.go

```diff
@@ -332,6 +359,111 @@ func createRun(opts *CreateOptions) error {
 	return nil
 }
 
+func interactiveGitIgnoreLicense(client *api.Client, hostname string) (string, string, error) {
+
+	var addBoth bool
+	var initialQs []*survey.Question
+
+	addGitIgnoreLicense := &survey.Question{
+		Name: "addGitIgnoreLicense",
+		Prompt: &survey.Confirm{
+			Message: "Would you like to add a .gitignore or a license?",
```

Comment:
Having played around with this a fair bit at this point, I feel like a better approach would be to split these out:

```
Would you like to add a .gitignore? y/N
  (if y, do the ignore select)
Would you like to add a license? y/N
  (if y, do the license select)
```

The confirm + multi select + up to two selects started to feel a bit cumbersome in my testing.